### PR TITLE
chore: Add guidelines draft

### PR DESCRIPTION
**Pull request by IagoLast**
Please give feedback and lets iterate from here

### CODE REVIEW

File: guidelines/monthly.md [hunk 1]
```diff
@@ -1 +1,26 @@
...
+
+About
+
+- BarcelonaJS will celebrate one event per month
+- The event is hosted by an external entity/company
+- The event should start around 19:00
+- The event must be free
+- BarcelonaJS won't charge anything for organizing the event.
+- All participants and organizers should agree with our CoC
+- The external entity can do an intro talk limited to 5m
```
Comment (IagoLast) [inline guidelines/monthly.md:11]: The idea here is to limit marketing actions from companies

File: guidelines/monthly.md [hunk 2]
```diff
@@ -1 +1,26 @@
...
+
+Talks:
+
+- Talks must be in English
+- Talks should be in the 20-90m range.
+- Talks should be javascript related.
+
+Required information:
```
Comment (IagoLast) [inline guidelines/monthly.md:19]: Not sure if this belongs here

File: guidelines/monthly.md [hunk 3]
```diff
@@ -1 +1,26 @@
...
+- The event must be free
+- BarcelonaJS won't charge anything for organizing the event.
+- All participants and organizers should agree with our CoC
+- The external entity can do an intro talk limited to 5m
+
+Talks:
+
+- Talks must be in English
+- Talks should be in the 20-90m range.
+- Talks should be javascript related.
```
Comment (motanelu) [inline guidelines/monthly.md]: I would rephrase this to javascript / frontend related. We had in the past talks about CSS / SaSS.

File: guidelines/monthly.md [hunk 4]
```diff
@@ -1 +1,26 @@
...
+- The event should start around 19:00
+- The event must be free
+- BarcelonaJS won't charge anything for organizing the event.
+- All participants and organizers should agree with our CoC
+- The external entity can do an intro talk limited to 5m
+
+Talks:
+
+- Talks must be in English
+- Talks should be in the 20-90m range.
```
Comment (onedesert) [inline guidelines/monthly.md]: 90m?

File: guidelines/monthly.md (hunk 2 above)
Comment (onedesert) [reply guidelines/monthly.md:19]: I think it does belong to here, I would push some timeframes (see proposals)

File: guidelines/monthly.md [hunk 5]
```diff
@@ -1 +1,26 @@
...
+- Talks should be javascript related.
+
+Required information:
+
+- Company must provide company name
+- Company must provide venue address.
```
Comment (onedesert) [inline guidelines/monthly.md:23]:
```suggestion
- Company must provide company name (3 weeks before the event)
- Company must provide venue address. (3 weeks before the event)
```
